dspace-api/src/main/java/org/dspace/app/requestitem/RequestItemAuthor.java

```diff
@@ -38,4 +41,29 @@ public String getEmail() {
     public String getFullName() {
         return fullName;
     }
+    /**
+     * Build a comma-list of addresses from a list of request recipients.
+     * @param recipients those to receive the request.
+     * @return addresses of the recipients, separated by ", ".
+     */
+    public static String listAddresses(List<RequestItemAuthor> recipients) {
+        List<String> addresses = new ArrayList(recipients.size());
+        for (RequestItemAuthor recipient : recipients) {
+            addresses.add(recipient.getEmail());
+        }
+        return String.join(", ", addresses);
+    }
+
+    /**
+     * Build a comma-list of full names from a list of request recipients.
+     * @param recipients those to receive the request.
+     * @return names of the recipients, separated by ", ".
+     */
+    public static String listNames(List<RequestItemAuthor> recipients) {
```

Comment:
this method is not used

Reply:
True.  I don't know why it was added.  I've removed it.